test/e2e/install/install_test.go

```diff
@@ -19,6 +19,7 @@ package install
 import (
 	"context"
 	"fmt"
+	"github.com/openshift-kni/numaresources-operator/test/e2e/label"
```

Comment:
misplaced import

Reply:
true and it's annoying, but I don't think is worth waiting more to fix this. I can send a cleanup later.
There's a proposal about adding a CI check for imports, but IIRC requires more work.